### DISCUSSION TITLE
Rotation screen

### DISCUSSION THREAD

**Discussion by alfreddo56** [agronick/aa-torque#125] [General]
hi, since I use aatorque with performance monitor on an android car radio and it works, is it possible to rotate the three indicators so as to see them horizontally? because otherwise I can't use it. Thanks

**Comment by agronick** [comment]
Yeah, I can look into making an option to run it as a standalone app.

**Reply by alfreddo56** [reply]
Thank you

**Comment by Braddly** [comment]
Hello, how can I rotate the display? Currently, I only have a vertical display in my car.
